tests/test_iam.py

```diff
@@ -181,7 +181,7 @@ def test_iam_role_unused(self):
             'resource': 'iam-role',
             'filters': ['unused']}, session_factory=session_factory)
         resources = p.run()
-        self.assertEqual(len(resources), 7)
+        self.assertEqual(len(resources), 6)
```

Comment:
After adding two new roles to the IAM fixture, one used, and one not, I expected this value to change to 8. It instead change to 9, which lead me to discover issue 2.